code/game/turfs/simulated/floor/plating.dm

```diff
@@ -159,18 +159,38 @@
 	nitrogen = 0
 	temperature = TCMB
 
+/turf/simulated/floor/plating/abductor
+	name = "alien floor"
+	icon_state = "alienpod1"
+
+/turf/simulated/floor/plating/abductor/New()
+	..()
+	icon_state = "alienpod[rand(1,9)]"
+
 /turf/simulated/floor/plating/lava
+	name = "lava"
 	icon_state = "lava"
+	baseturf = /turf/simulated/floor/plating/lava //lava all the way down
+	slowdown = 2
 
 /turf/simulated/floor/plating/lava/airless
 	oxygen = 0
 	nitrogen = 0
 	temperature = TCMB
 
-/turf/simulated/floor/plating/abductor
-	name = "alien floor"
-	icon_state = "alienpod1"
-
-/turf/simulated/floor/plating/abductor/New()
-	..()
-	icon_state = "alienpod[rand(1,9)]"
\ No newline at end of file
+/turf/simulated/floor/plating/lava/Entered(atom/movable/AM)

```

Comment:
I could do that. I figured spamming resist and meds on yourself for the privilege of saying "haha I'm on lava" wouldnt be the worst thing in the world though. Back during google code there was a big push to cut down on processing objects so I am hesitant to use them (even if they're only temporary).